### PR TITLE
chore: metadata improvement

### PR DESCRIPTION
**Pull request by jpgonzalezra**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/main.rs [hunk 1]
```diff
@@ -365,25 +352,26 @@ fn generate_image_and_metadata(
     let mut combined_image = ImageBuffer::new(width, height);
     // kill me now
```
Comment (ghost) [inline src/main.rs]: 🙄 mi marca de agua se puede sacar aca

File: src/main.rs [hunk 2]
```diff
@@ -365,15 +365,15 @@ fn generate_image_and_metadata(
         }
 
         combined_image
-            .save(format!("./{}/{}.png", output_path, image_name))
+            .save(format!("{}/{}.png", output_path, image_name))
```
Comment (ghost) [inline src/main.rs:367]: let file_path = format!("{}/{}.png", output_path, image_name);

....save(file_path.clone())

let mut file = File::create(file_path.clone()).unwrap();

**Review by ghost — APPROVED** [review]

File: src/main.rs (hunk 2 above)
Comment (jpgonzalezra) [reply src/main.rs:367]: ufff my bad

Comment (jpgonzalezra) [reply src/main.rs:367]: i will fix it in another PR

File: src/main.rs (hunk 1 above)
Comment (jpgonzalezra) [reply src/main.rs]: sorry, let me do it ^^

**Pull request merged**